README.md

```diff
@@ -10,6 +10,27 @@ You can install the command globally using:
 npm i -g @projectsophon/df-plugin-dev-server
 ```
 
+## Docker image installation and running
```

Comment:
Can you move this section further down the page? I still think the npm workflow should be the default recommended with docker as the backup workflow.

Reply:
Sure, moved. Added subsection for Docker to inform people that networking with docker is hard and it is easy to give unwanted access to containers. All described in `Security notes for docker containers`